resources/strings.js

```diff
@@ -399,9 +399,9 @@ const schedule = [
   {id: "crypticChaosIILunch", title: "Cryptic Chaos II", day: "2", time: "12:10pm - 12:55pm", location: "Fields"},
   {id: "lunchII", title: "Lunch II", day: "2", time: "1:00pm - 1:45pm", location: "Dining Hall"},
   {id: "crypticChaosILunch", title: "Cryptic Chaos I", day: "2", time: "1:00pm - 1:45pm", location: "Fields"},
-  {id: "workshopSessionIII", title: "Workshop Session III  >", day: "2", time: "1:55pm - 2:45pm", location: ""},
-  {id: "workshopSessionIV", title: "Workshop Session IV  >", day: "2", time: "2:55pm - 3:45pm", location: ""},
-  {id: "workshopSessionV", title: "Workshop Session V  >", day: "2", time: "3:55pm - 4:45pm", location: ""},
+  {id: "workshopSessionIII", title: "Workshop Session III", day: "2", time: "1:55pm - 2:45pm", location: ""},
+  {id: "workshopSessionIV", title: "Workshop Session IV", day: "2", time: "2:55pm - 3:45pm", location: ""},
+  {id: "workshopSessionV", title: "Workshop Session V", day: "2", time: "3:55pm - 4:45pm", location: ""},
```

Comment:
Add "Various Locations" or something to location. Same with Day 2 workshop sessions